src/main/java/org/snakeyaml/engine/v2/common/FlowStyle.kt

```diff
@@ -18,18 +18,18 @@ package org.snakeyaml.engine.v2.common
  * styles rely on explicit indicators to denote nesting and scope.
  */
 enum class FlowStyle {
-  /**
-   * Flow style
-   */
-  FLOW,
+    /**
+     * Flow style
+     */
+    FLOW,
```

Comment:
Looks like accidental formatting change, but I don't want you to be blocked on this. Later on we can configure e. g. ktlint.

Reply:
I think I converted this file before the EditorConfig was added, and it was never updated.

And yeah, adding a better way of formatting would be nice. I'm not a big fan of ktlint, but EditorConfig doesn't seem to work as well as I remember. At least committing the `.idea/codeStyles/` would help.